demos/twa-webview-fallback/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You need licenses and such here (and probably other files too).

I would be nice if we could highlight (maybe just in a comment at the top) what's special about this AndroidManifest (that it's the standard one plus WebViewFallbackActivity).